src/_data/LiveData.js

```diff
@@ -43,6 +43,13 @@ export const changeLanguage = langCode => {
     api.getTradingTimes(new Date());
 };
 
+export const getTickHistory = async (symbol) => {
```

Comment:
@qingweibinary  i don't think thats necessary. The method does change a state in any way beside making a call to an WS. HAd it been we are updating or changing a state example let's say logging history or something then would have made sense. I think it should be ok like this just like how we handle the one above it.